x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/exception_item_card/meta.tsx

```diff
@@ -0,0 +1,161 @@
+import { SecuritySolutionLinkAnchor } from '../../../../common/components/links';
+import { RuleDetailTabs } from '../../../../detections/pages/detection_engine/rules/details';
+import { getRuleDetailsTabUrl } from '../../../../common/components/link_to/redirect_to_detection_engine';
+
+const StyledFlexItem = styled(EuiFlexItem)`
+  border-right: 1px solid #d3dae6;
+  padding: 4px 12px 4px 0;
+`;
+
+export interface ExceptionItemCardMetaInfoProps {
+  item: ExceptionListItemSchema;
+  references: RuleReferenceSchema[];
+  dataTestSubj: string;
+}
+
+export const ExceptionItemCardMetaInfo = memo<ExceptionItemCardMetaInfoProps>(
+  ({ item, references, dataTestSubj }) => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+    const onAffectedRulesClick = () => setIsPopoverOpen((isOpen) => !isOpen);
+    const onClosePopover = () => setIsPopoverOpen(false);
+
+    const itemActions = useMemo((): EuiContextMenuPanelProps['items'] => {
+      if (references == null) {
+        return [];
+      }
+      return references.map((reference) => (
+        <EuiContextMenuItem
+          data-test-subj={`${dataTestSubj}-actionItem-${reference.id}`}
+          key={reference.id}
+        >
+          <EuiToolTip content={reference.name} anchorClassName="eui-textTruncate">
+            <SecuritySolutionLinkAnchor
+              data-test-subj="ruleName"
+              deepLinkId={SecurityPageName.rules}
+              path={getRuleDetailsTabUrl(reference.id, RuleDetailTabs.alerts)}
+            >
+              {reference.name}
+            </SecuritySolutionLinkAnchor>
+          </EuiToolTip>
+        </EuiContextMenuItem>
+      ));
+    }, [references, dataTestSubj]);
+
+    return (
+      <EuiFlexGroup
+        alignItems="center"
+        responsive={false}
+        gutterSize="s"
+        data-test-subj={dataTestSubj}
+      >
+        <StyledFlexItem grow={false}>
+          <MetaInfoDetails
+            fieldName="created_by"
+            label={i18n.EXCEPTION_ITEM_CREATED_LABEL}
+            value1={<FormattedDate fieldName="created_at" value={item.created_at} />}
+            value2={item.created_by}
+            dataTestSubj={`${dataTestSubj}-createdBy`}
+          />
+        </StyledFlexItem>
+        <StyledFlexItem grow={false}>
+          <MetaInfoDetails
+            fieldName="updated_by"
+            label={i18n.EXCEPTION_ITEM_UPDATED_LABEL}
+            value1={<FormattedDate fieldName="updated_at" value={item.updated_at} />}
+            value2={item.updated_by}
+            dataTestSubj={`${dataTestSubj}-updatedBy`}
+          />
+        </StyledFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiPopover
+            button={
+              <EuiButtonEmpty
+                onClick={onAffectedRulesClick}
+                iconType="list"
+                data-test-subj={`${dataTestSubj}-affectedRulesButton`}
+              >
+                {i18n.AFFECTED_RULES(references.length)}
```

Comment:
Pulled down and tested and ran into an error where references sometimes was null. Once I fixed this on my local everything seemed to work pretty smoothly!

Reply:
Fixed!